### ISSUE TITLE
pkgconf: Version is missing in v0.12.0

### ISSUE THREAD

**Issue by alltilla** [alanxz/rabbitmq-c#755]
With `v0.12.0`, the generated `librabbitmq.pc` does not have any value for `Version`. This makes version checks fail with `pkg-config`.

### `v0.11.0`
```
prefix=/usr/local
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include

Name: rabbitmq-c
Description: An AMQP 0-9-1 client library
Version: 0.11.0
URL: https://github.com/alanxz/rabbitmq-c
Requires.private: openssl
Libs: -L${libdir} -lrabbitmq
Libs.private:  -lrt 
CFlags: -I${includedir}
```

### `v0.12.0`
```
prefix=/usr/local
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include

Name: rabbitmq-c
Description: An AMQP 0-9-1 client library
Version: 
URL: https://github.com/alanxz/rabbitmq-c
Requires.private: 
Libs: -L${libdir} -lrabbitmq
Libs.private:  -lrt -lssl -lcrypto 
Cflags: -I${includedir}
```

### Reproduction
<details>

```
$ git checkout v0.12.0 && cmake . && make && sudo make install && cat /usr/local/lib/pkgconfig/librabbitmq.pc
-- The C compiler identification is GNU 11.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Setting build type to 'Release' as none was specified.
-- Looking for getaddrinfo
-- Looking for getaddrinfo - found
-- Looking for socket
-- Looking for socket - found
-- Looking for poll
-- Looking for poll - found
-- Looking for clock_gettime in rt
-- Looking for clock_gettime in rt - found
-- Looking for posix_spawnp in rt
-- Looking for posix_spawnp in rt - found
-- Found OpenSSL: /usr/lib/x86_64-linux-gnu/libcrypto.so (found suitable version "3.0.2", minimum required is "1.1.1")  
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Failed
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
-- Building rabbitmq as a shared library - yes
-- Building rabbitmq as a static library - yes
-- Configuring done
-- Generating done
-- Build files have been written to: /home/alltilla/Repos/rabbitmq-c
[  2%] Building C object librabbitmq/CMakeFiles/rabbitmq.dir/amqp_api.c.o
[  5%] Building C object librabbitmq/CMakeFiles/rabbitmq.dir/amqp_connection.c.o
[  7%] Building C object librabbitmq/CMakeFiles/rabbitmq.dir/amqp_consumer.c.o
[ 10%] Building C object librabbitmq/CMakeFiles/rabbitmq.dir/amqp_framing.c.o
[ 13%] Building C object librabbitmq/CMakeFiles/rabbitmq.dir/amqp_mem.c.o
[ 15%] Building C object librabbitmq/CMakeFiles/rabbitmq.dir/amqp_openssl.c.o
[ 18%] Building C object librabbitmq/CMakeFiles/rabbitmq.dir/amqp_openssl_bio.c.o
[ 21%] Building C object librabbitmq/CMakeFiles/rabbitmq.dir/amqp_socket.c.o
[ 23%] Building C object librabbitmq/CMakeFiles/rabbitmq.dir/amqp_table.c.o
[ 26%] Building C object librabbitmq/CMakeFiles/rabbitmq.dir/amqp_tcp_socket.c.o
[ 28%] Building C object librabbitmq/CMakeFiles/rabbitmq.dir/amqp_time.c.o
[ 31%] Building C object librabbitmq/CMakeFiles/rabbitmq.dir/amqp_url.c.o
[ 34%] Linking C shared library librabbitmq.so
[ 34%] Built target rabbitmq
[ 36%] Building C object librabbitmq/CMakeFiles/rabbitmq-static.dir/amqp_api.c.o
[ 39%] Building C object librabbitmq/CMakeFiles/rabbitmq-static.dir/amqp_connection.c.o
[ 42%] Building C object librabbitmq/CMakeFiles/rabbitmq-static.dir/amqp_consumer.c.o
[ 44%] Building C object librabbitmq/CMakeFiles/rabbitmq-static.dir/amqp_framing.c.o
[ 47%] Building C object librabbitmq/CMakeFiles/rabbitmq-static.dir/amqp_mem.c.o
[ 50%] Building C object librabbitmq/CMakeFiles/rabbitmq-static.dir/amqp_openssl.c.o
[ 52%] Building C object librabbitmq/CMakeFiles/rabbitmq-static.dir/amqp_openssl_bio.c.o
[ 55%] Building C object librabbitmq/CMakeFiles/rabbitmq-static.dir/amqp_socket.c.o
[ 57%] Building C object librabbitmq/CMakeFiles/rabbitmq-static.dir/amqp_table.c.o
[ 60%] Building C object librabbitmq/CMakeFiles/rabbitmq-static.dir/amqp_tcp_socket.c.o
[ 63%] Building C object librabbitmq/CMakeFiles/rabbitmq-static.dir/amqp_time.c.o
[ 65%] Building C object librabbitmq/CMakeFiles/rabbitmq-static.dir/amqp_url.c.o
[ 68%] Linking C static library librabbitmq.a
[ 68%] Built target rabbitmq-static
[ 71%] Building C object tests/CMakeFiles/test_parse_url.dir/test_parse_url.c.o
[ 73%] Linking C executable test_parse_url
[ 73%] Built target test_parse_url
[ 76%] Building C object tests/CMakeFiles/test_tables.dir/test_tables.c.o
[ 78%] Linking C executable test_tables
[ 78%] Built target test_tables
[ 81%] Building C object tests/CMakeFiles/test_status_enum.dir/test_status_enum.c.o
[ 84%] Linking C executable test_status_enum
[ 84%] Built target test_status_enum
[ 86%] Building C object tests/CMakeFiles/test_basic.dir/test_basic.c.o
[ 89%] Linking C executable test_basic
[ 89%] Built target test_basic
[ 92%] Building C object tests/CMakeFiles/test_sasl_mechanism.dir/test_sasl_mechanism.c.o
[ 94%] Linking C executable test_sasl_mechanism
[ 94%] Built target test_sasl_mechanism
[ 97%] Building C object tests/CMakeFiles/test_merge_capabilities.dir/test_merge_capabilities.c.o
[100%] Linking C executable test_merge_capabilities
[100%] Built target test_merge_capabilities
Consolidate compiler generated dependencies of target rabbitmq
[ 34%] Built target rabbitmq
Consolidate compiler generated dependencies of target rabbitmq-static
[ 68%] Built target rabbitmq-static
Consolidate compiler generated dependencies of target test_parse_url
[ 73%] Built target test_parse_url
Consolidate compiler generated dependencies of target test_tables
[ 78%] Built target test_tables
Consolidate compiler generated dependencies of target test_status_enum
[ 84%] Built target test_status_enum
Consolidate compiler generated dependencies of target test_basic
[ 89%] Built target test_basic
Consolidate compiler generated dependencies of target test_sasl_mechanism
[ 94%] Built target test_sasl_mechanism
Consolidate compiler generated dependencies of target test_merge_capabilities
[100%] Built target test_merge_capabilities
Install the project...
-- Install configuration: "Release"
-- Installing: /usr/local/lib/librabbitmq.so.0.12.0
-- Up-to-date: /usr/local/lib/librabbitmq.so.4
-- Set runtime path of "/usr/local/lib/librabbitmq.so.0.12.0" to "$ORIGIN"
-- Up-to-date: /usr/local/lib/librabbitmq.so
-- Installing: /usr/local/lib/librabbitmq.a
-- Up-to-date: /usr/local/include/amqp.h
-- Up-to-date: /usr/local/include/amqp_framing.h
-- Up-to-date: /usr/local/include/amqp_tcp_socket.h
-- Up-to-date: /usr/local/include/amqp_ssl_socket.h
-- Installing: /usr/local/include/rabbitmq-c/amqp.h
-- Up-to-date: /usr/local/include/rabbitmq-c/framing.h
-- Up-to-date: /usr/local/include/rabbitmq-c/tcp_socket.h
-- Installing: /usr/local/include/rabbitmq-c/ssl_socket.h
-- Installing: /usr/local/include/rabbitmq-c/export.h
-- Installing: /usr/local/lib/cmake/rabbitmq-c/rabbitmq-c-config.cmake
-- Installing: /usr/local/lib/cmake/rabbitmq-c/rabbitmq-c-config-version.cmake
-- Installing: /usr/local/lib/cmake/rabbitmq-c/rabbitmq-targets.cmake
-- Installing: /usr/local/lib/cmake/rabbitmq-c/rabbitmq-targets-release.cmake
-- Installing: /usr/local/lib/pkgconfig/librabbitmq.pc
prefix=/usr/local
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include

Name: rabbitmq-c
Description: An AMQP 0-9-1 client library
Version: 
URL: https://github.com/alanxz/rabbitmq-c
Requires.private: 
Libs: -L${libdir} -lrabbitmq
Libs.private:  -lrt -lssl -lcrypto 
Cflags: -I${includedir}
```

</details>

**Comment by alanxz** [comment]
This was fixed in #751. I intend on creating another release (v0.13.0) shortly that should fix this.

**Comment by alltilla** [comment]
Awesome, thank you! :)
